### PR TITLE
[docs] Readme updated

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
/cc @inikulin @VasilyStrelyaev 

I've added info from the landing page and create common sections: license, contributing... 
I haven't added info from Getting Started, there will be a link to the Getting Started section from the documentation.

Some notes:
1. Do we need some introduction about TestCafe or "Modern functional testing framework" is enough?
2. Do we need the **Modules** section?
3. In the **Documentation** section, there will be links to the main sections on gh-pages. They do not work yet.
4. The image for the **Descriptive Reports** section isn't ready.

### CODE REVIEW

**Comment by inikulin** [comment]
\cc @DevExpress/testcafe-docs

**Comment by inikulin** [comment]
> 1. Do we need some introduction about TestCafe or "Modern functional testing framework" is enough?

Yes

> 2. Do we need the Modules section?

I think we can get rid of it

> 3. In the Documentation section, there will be links to the main sections on gh-pages. They do not work yet.

Let's publish gh-pages

**Comment by inikulin** [comment]
BTW, since README includes landing page + getting started maybe it makes sense to write gulp script which just basically take these sections from docs and insert it into README, so they will be consistent? \cc @VasilyStrelyaev

**Comment by inikulin** [comment]
BTW, we also should also run documentation tests for README.

**Comment by inikulin** [comment]
And I have a question to @VasilyStrelyaev: if we have external link in docs (e.g. to node website) do we just check this link or we crawling node website as well?

**Comment by VasilyStrelyaev** [comment]
we crawl everything

**Comment by inikulin** [comment]
> we crawl everything

Not cool. We need some domain filter then

**Comment by kirovboris** [comment]
Markdown is broken
https://github.com/MargaritaLoseva/testcafe/blob/208f5075f15c61b73e91c6eb0a0056ca92255e02/README.md

**Comment by MargaritaLoseva** [comment]
@kirovboris Yes, it's my mistake, I'll repair this.

**Comment by VasilyStrelyaev** [comment]
> Not cool. We need some domain filter then

What's wrong with that? I thought it's great if we know that an id we refer to no longer exists.

**Comment by inikulin** [comment]
@VasilyStrelyaev seems like you dont't get my question. If we have link to nodejs.org, do we just check that nodejs.org is available and stop this crawling branch or we continue crawling links on nodejs.org?

**Comment by VasilyStrelyaev** [comment]
Oh, no, of course we don't crawl external websites. We check that the page is available and then if the link has a hash, we parse that single page and check if the id exists.

**Comment by inikulin** [comment]
@VasilyStrelyaev :+1:

**Comment by VasilyStrelyaev** [comment]
> BTW, since README includes landing page + getting started maybe it makes sense to write gulp script which just basically take these sections from docs and insert it into README, so they will be consistent? \cc @VasilyStrelyaev

Well, the landing page is not Markdown but HTML...

**Comment by inikulin** [comment]
Maybe we should place feature list above getting started? Also, we need table of contents at the top of the page

File: README.md [hunk 1]
```diff
@@ -4,37 +4,180 @@
...
+
+- [Getting Started](http://devexpress.github.io/testcafe/documentation/getting-started/)
+- [Test API](http://devexpress.github.io/testcafe/documentation/test-api/)
+- [Using TestCafe](http://devexpress.github.io/testcafe/documentation/using-testcafe/)
+- [Extending TestCafe](http://devexpress.github.io/testcafe/documentation/extending-testcafe/)
+- [Recipes](http://devexpress.github.io/testcafe/documentation/recipes/)
+
+# Contributing
+
+Please use our issues page to report a bug or request a feature.
```
Comment (inikulin) [inline README.md]: Add link to issues page

File: README.md [hunk 2]
```diff
@@ -4,37 +4,180 @@
...
+
+For more information on how to help us improve TestCafe, please see the Contributing.md file.
```
Comment (inikulin) [inline README.md]: Add link to contributing.md

File: README.md [hunk 3]
```diff
@@ -4,37 +4,180 @@
...
+
+# License
+
+See the License.md file.
```
Comment (inikulin) [inline README.md]: Just write `MIT` here

File: README.md [hunk 4]
```diff
@@ -4,37 +4,180 @@
...
 </p>
 <p align="center">
-<i>Modern functional testing framework.</i>
+<i>Modern functional testing framework</i>
 </p>
 
 
 ----
 
+TestCafe is a simple and powerful framework for testing websites and apps.
```
Comment (inikulin) [inline README.md]: We need link to website in header

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

File: README.md [hunk 5]
```diff
@@ -4,37 +4,180 @@
     </a>
 </p>
 <p align="center">
-<i>Modern functional testing framework.</i>
+<i>Modern functional testing framework</i>
```
Comment (inikulin) [inline README.md]: Change to `Automated browser testing for modern web development stack.`

File: README.md (hunk 2 above)
Comment (MargaritaLoseva) [inline README.md]: There is no Contributin.md file yet. Should we create it?

Comment (MargaritaLoseva) [inline README.md]: @inikulin  There is no Contributing.md file yet. Should we create it?

Comment (inikulin) [inline README.md]: It will be created in context of https://github.com/DevExpress/testcafe/issues/703 I guess

Comment (inikulin) [inline README.md]: Just add link to CONTRIBUTING.md in the root of the repo

**Comment by inikulin** [comment]
We need to add npm version badge under the title as well, for example see https://github.com/inikulin/parse5/blob/master/README.md

File: README.md [hunk 6]
```diff
@@ -1,40 +1,206 @@
...
 <p align="center">
-<i>Modern functional testing framework.</i>
+<i>Automated browser testing for modern web development stack.</i>
 </p>
 
-
 ----
 
+TestCafe is a simple and powerful framework for testing websites and apps.
+It allows you to easily create, maintain and execute automated web tests across browsers, operating systems and devices.
```
Comment (inikulin) [inline README.md]: `testing websites and apps` -> `testing websites and web apps`

File: README.md [hunk 7]
```diff
@@ -1,40 +1,206 @@
...
+testcafe safari tests/
+```
+
+TestCafe aggregates test results from different browsers and outputs them into one comprehensive report.
+
+## Write Test Code Using ES2016
+
+You can write TestCafe tests in ES2016 using all latest JavaScript features like 'async/await'.
+
+TestCafe introduces simple, but powerful test API. It offers a couple of dozen methods covering all user actions. Chained syntax produces code that is easy to write and read. Furthermore, you are free to use any assertion library to perform verifications of different kinds.
```
Comment (inikulin) [inline README.md]: `covering all user actions` - I'm not sure if we cover `all` actions

File: README.md [hunk 8]
```diff
@@ -1,40 +1,206 @@
...
+
+```bash
+testcafe safari tests/
+```
+
+TestCafe aggregates test results from different browsers and outputs them into one comprehensive report.
+
+## Write Test Code Using ES2016
+
+You can write TestCafe tests in ES2016 using all latest JavaScript features like 'async/await'.
```
Comment (inikulin) [inline README.md]: Highlight `async/await` as code

File: README.md [hunk 9]
```diff
@@ -1,40 +1,206 @@
...
+- [Extending TestCafe](http://devexpress.github.io/testcafe/documentation/extending-testcafe/)
+- [Recipes](http://devexpress.github.io/testcafe/documentation/recipes/)
+
+# Contributing
 
- - [testcafe-hammerhead](https://github.com/DevExpress/testcafe-hammerhead) - the heart.
- - [testcafe-browser-natives](https://github.com/DevExpress/testcafe-browser-natives) - the library for performing platform-dependent actions on browser windows.
- - [testcafe-legacy-api](https://github.com/DevExpress/testcafe-legacy-api) - the legacy API support.
- - [testcafe-gh-page-assets](https://github.com/DevExpress/testcafe-gh-page-assets) - the website assets.
+Please use our [issues page](https://github.com/DevExpress/testcafe/issues) to report a bug or request a feature.
```
Comment (inikulin) [inline README.md]: For general purpose questions and discussions use https://testcafe-discuss.devexpress.com/

File: README.md (hunk 4 above)
Comment (inikulin) [inline README.md]: @MargaritaLoseva link is still missing

File: README.md [hunk 10]
```diff
@@ -1,40 +1,206 @@
...
 
-#### Built-in reporters
- - [testcafe-reporter-spec](https://github.com/DevExpress/testcafe-reporter-spec) - the "Spec" reporter plugin.
- - [testcafe-reporter-list](https://github.com/DevExpress/testcafe-reporter-list) - the "List" reporter plugin.
- - [testcafe-reporter-minimal](https://github.com/DevExpress/testcafe-reporter-minimal) - the "Minimal" reporter plugin.
- - [testcafe-reporter-json](https://github.com/DevExpress/testcafe-reporter-json) - the "JSON" reporter plugin.
- - [testcafe-reporter-xunit](https://github.com/DevExpress/testcafe-reporter-xunit) - the "xUnit" reporter plugin.
+For more information on how to help us improve TestCafe, please see the [CONTRIBUTING](CONTRIBUTING.md) file.
```
Comment (inikulin) [inline README.md]: `CONTRIBUTING.md` file

**Comment by MargaritaLoseva** [comment]
FPR

@inikulin  The current npm version badge shows v. 15.1.3. Is it normal?

**Comment by inikulin** [comment]
Yeah, it's just fine for now. Once we publish version with "latest" tag - it will be replaces

File: README.md [hunk 11]
```diff
@@ -1,40 +1,212 @@
 <p align="center">
-    <a href="https://github.com/DevExpress/testcafe">
+    <a href="http://devexpress.github.io/testcafe">
```
Comment (inikulin) [inline README.md]: Can we have link below the logo as a regular link?

**Comment by inikulin** [comment]
Overall lgtm, just add link as centered text under the logo

**Comment by inikulin** [comment]
lgtm

\cc @DevExpress/testcafe-docs

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**